app/models/view/alerts/Alert.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.view.alerts;
+
+import com.arpnetworking.logback.annotations.Loggable;
+import com.google.common.collect.ImmutableMap;
+import models.internal.alerts.AlertEvaluationResult;
+import models.internal.scheduling.JobExecution;
+
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * View model for an alert.
+ * <p>
+ * Play view models are mutable.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ */
+@Loggable
+public final class Alert {
+    private UUID _id;
```

Comment:
Is the pattern now to type these more explicitly. I believe the API treats these as opaque strings.

Reply:
This is another case where I went with what reports did.

It's a UUID on the [report model](https://github.com/ArpNetworking/metrics-portal/blob/master/app/models/view/reports/Report.java#L40) and [on the routes](https://github.com/ArpNetworking/metrics-portal/blob/master/conf/portal.routes#L39).

I'm not too opinionated here since IMO the versioning on the endpoint and the decoupling of the view model give us sufficient wiggle room were this ever to change.

Reply:
I noticed that afterwards as well. The other routes are string. Pretty sure I just missed this on the reporting stuff. Not a big deal, maybe file an issue.